Sources/XCBuildSupport/XcodeBuildSystem.swift

```diff
@@ -104,10 +104,13 @@ public final class XcodeBuildSystem: SPMBuildCore.BuildSystem {
         arguments += buildParameters.xcbuildFlags
 
         let delegate = createBuildDelegate()
+        
         let redirection: Process.OutputRedirection = .stream(stdout: delegate.parse(bytes:), stderr: { bytes in
-            self.diagnostics.emit(StringError(String(bytes: bytes, encoding: .utf8)!))
+            if let str = String(bytes: bytes, encoding: .utf8) {
```

Comment:
Also, is it the right thing to emit all of this output as warnings?  If I have an error that is emitted, would it show up as parseable diagnostics as well as on stderr, and thus show up as a warning as well as an error?

Reply:
That's what the report in the radar seems to suggest. @jakepetroules Could you confirm the above question?  Or would we need to filter and treat certain stderr as actual errors?

Reply:
All XCBuild output is communicated as structured (JSON) data via stdout, if we emit anything via stderr it's unintentional and can be dropped.